src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator.UnitTests/Resources/SingleString_EmitFormatMethodsVisualBasicAsync_True/Resources.resx.Error.vb

```diff
@@ -0,0 +1,4 @@
+﻿#error System.NotImplementedException: The method or operation is not implemented.
+#error    at Microsoft.CodeAnalysis.ResxSourceGenerator.AbstractResxGenerator.Impl.Execute(CancellationToken cancellationToken) in C:\dev\roslyn-analyzers\src\Microsoft.CodeAnalysis.ResxSourceGenerator\Microsoft.CodeAnalysis.ResxSourceGenerator\AbstractResxGenerator.cs:line 328
+#error    at Microsoft.CodeAnalysis.ResxSourceGenerator.AbstractResxGenerator.ProcessResourceFile(SourceProductionContext context, ResourceInformation resourceFile) in C:\dev\roslyn-analyzers\src\Microsoft.CodeAnalysis.ResxSourceGenerator\Microsoft.CodeAnalysis.ResxSourceGenerator\AbstractResxGenerator.cs:line 166
+#error    at Microsoft.CodeAnalysis.ResxSourceGenerator.AbstractResxGenerator.<>c.<Initialize>b__1_3(SourceProductionContext context, ResourceInformation resourceInformation) in C:\dev\roslyn-analyzers\src\Microsoft.CodeAnalysis.ResxSourceGenerator\Microsoft.CodeAnalysis.ResxSourceGenerator\AbstractResxGenerator.cs:line 117
```

Comment:
I don't quite understand this?

Reply:
In the event an error occurs during generation, the exception message is printed to the output in this form.

Reply:
@sharwell But the test is skipped, why are we producing these files?

and if exceptions are to be tested this way, they will pass/fail depending on the repository path on the machine since that is part of the stack trace.

Reply:
> But the test is [skipped](https://github.com/dotnet/roslyn-analyzers/blob/c157d84e84982d3a03ebd0c5bb4c3d6c927327e4/src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator.UnitTests/ResxGeneratorTests.cs#L520), why are we producing these files?

We aren't. I manually enabled the test to force the creation of this file as an example of why it's failing. When it's eventually fixed, this will make the diff clearer.